src/schema/v2/homeView/sectionTypes/QuickLinks.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { NodeInterface } from "../../object_identification"
+import { HomeViewGenericSectionInterface } from "./GenericSectionInterface"
+import { HomeViewSectionTypeNames } from "./names"
+import { standardSectionFields } from "./GenericSectionInterface"
+import { QuickLink } from "../sections/QuickLinks"
+
+const QuickLinkType = new GraphQLObjectType<QuickLink, ResolverContext>({
+  name: "QuickLink",
+  fields: () => ({
+    title: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Quick link title",
+    },
+    href: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Quick link URL",
+    },
+    ownerType: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The context module for analytics",
+    },
+    contextScreenOwnerId: {
+      type: GraphQLString,
+      description: "The owner ID for the context module",
```

Comment:
@anandaroop @dblandin what do you think about this?

Reply:
What is this used for? The only specific value I see currently is "/collect?price_range=*-1000" which doesn't read as an ID to me.

Is this needed value need for the analytics support?

Reply:
yes, that's what we have in mind. We can also use the href as a payload for the tracking event. it's not clear so far.

Reply:
To capture the `href` value, you could use `destination_path` which we've included in similar events.

https://github.com/artsy/cohesion/pull/525